server/middleware/authorisationMiddleware.ts

```diff
@@ -14,7 +17,12 @@ export default function authorisationMiddleware(authorisedRoles: string[] = []):
         return res.redirect('/authError')
       }
 
-      return next()
+      return next(
+        rateLimiter({
+          windowMs: config.rateLimits.authorisation.windowTime,
+          maxRequests: config.rateLimits.authorisation.maxRequests,
+        }),
+      )
```

Comment:
This will just pass the middleware as a function to the error handler

I think this only compiles as next can take `any` as an error object